### PR TITLE
Fix: change current tab pannel due to library component

### PR DESCRIPTION
**Pull request by pkhlopiachyi**

### CODE REVIEW

File: src/components/Order/index.tsx [hunk 1]
```diff
@@ -130,6 +130,9 @@ export interface OrderComponentProps {
      */
     listenInputPrice?: () => void;
 }
+interface State {
```
Comment (josadcha) [inline src/components/Order/index.tsx]: add empty line between interfaces

File: src/components/TabPanel/index.tsx [hunk 2]
```diff
@@ -50,22 +50,30 @@ export interface TabPanelProps {
...
  */
-export class TabPanel extends React.Component<TabPanelProps> {
+export class TabPanel extends React.Component<TabPanelProps, TabPanelState> {
     public static defaultProps = {
         hideMode: HideMode.hide,
     };
 
+    public state = {
+        currentTab: 0,
+    };
```
Comment (josadcha) [inline src/components/TabPanel/index.tsx]: why did you add one more index if this component already has prop currentTabIndex?

File: src/components/TabPanel/index.tsx [hunk 3]
```diff
@@ -50,22 +50,30 @@ export interface TabPanelProps {
     /**
      * Current index of tab
      */
-    currentTabIndex: number;
+    currentTabIndex?: number;
```
Comment (josadcha) [inline src/components/TabPanel/index.tsx]: why did you make this property not necessary?

Comment (pkhlopiachyi) [reply src/components/TabPanel/index.tsx]: because we don't use it at create order component

File: src/components/Order/index.tsx [hunk 4]
```diff
@@ -139,7 +142,12 @@ const defaultOrderTypes: DropdownElem[] = [
 const splitBorder = 449;
 const defaultWidth = 635;
 
-class Order extends React.PureComponent<OrderComponentProps> {
+
+class Order extends React.PureComponent<OrderComponentProps, State> {
+    public state = {
+        index: 0,
+    };
```
Comment (josadcha) [inline src/components/Order/index.tsx]: tabIndex is the Prop of component/Orders soo you need to define index in containers/Orders state

**Comment by pkucherenk0** [comment]
Selection_321
Selection_322
